vueui/src/components/view/mcomputer/security/penalcontroller.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <div>
+  <div v-if="mechs.length > 0">
+  <hr>
+  <h3>Penal Mechs</h3>
+  <hr>
+    <div class="table">
+      <div class="header">
+        <div class="header-item">Pilot</div>
+        <div class="header-item">Mech Type</div>
+        <div class="header-item">Location</div>
+        <div class="header-item">Camera</div>
+        <div class="header-item">Lockdown</div>
+      </div>
+      <div class="mech" v-for="mech in mechs" :key="mech.ref">
+        <div class="item center">{{ mech.pilot }}</div>
+        <div class="item center">{{ mech.name }}</div>
+        <div class="item center">{{ mech.location }}</div>
+        <div class="item center"><vui-button :class="{'selected': camera_selected == 1}" :params="{track_mech: mech.ref}" :disabled="mech.camera == 0">Track</vui-button></div>
+        <div class="item center"><vui-button :class="{'red': mech.lockdown == 1}" :params="{lockdown_mech: mech.ref}">Lockdown</vui-button></div>
+      </div>
+    </div>
+    <div v-else>
+  	<h1>No Penal Mechs Detected</h1>
+    </div>
+	</div>
+  <div v-if="robots.length > 0">
+  <hr>
+  <h3>Penal Cyborgs</h3>
+  <hr>
+    <div class="table">
+      <div class="header">
+        <div class="header-item">Pilot</div>
+        <div class="header-item">Robot Type</div>
+        <div class="header-item">Location</div>
```

Comment:
ToDo here : Figure out how to do lockdown + camera for whatever mobtype these robots are supposed to be.